### PR TITLE
Increase the verbosity level of memquota to reduce log spam

### PR DESCRIPTION
**Pull request by ZackButcher**
**Release note**:

```release-note
None
```

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/1224)

### CODE REVIEW

**Comment by ZackButcher** [comment]
We do not, it's on my long list of things to do (rationalize all of our logging, and come up with some standards). But today we default to v=4 and those log messages are _incredibly_ uninteresting.

**Comment by ZackButcher** [comment]
In my mind, 0-2 = normal, 3-4 = debug, 5+ = verbose.

**Comment by ZackButcher** [comment]
/retest

Lets roll the dice...

**Comment by douglas-reid** [comment]
we should just force this in. let's forgo all pretext about this impacting e2e's here. every new PR that has been locally tested has felt this pain and has a bit to fix it:

@mandarjog ^^^

File: adapter/memquota2/dedup.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func (du *dedupUtil) reapDedup() {
 	du.oldDedup = du.recentDedup
 	du.recentDedup = t
 
-	if du.logger.VerbosityLevel(4) {
+	if du.logger.VerbosityLevel(6) {
 		du.logger.Infof("Running repear to reclaim %d old deduplication entries", len(t))
```
Comment (douglas-reid) [inline adapter/memquota2/dedup.go]: we might want to update "repear" to be `reaper`, while we are at it.

Comment (mandarjog) [reply adapter/memquota2/dedup.go]: Can we not log anything when len(t) is 0 ?

**Pull request merged**